embedded/EscapeROOM/radio_morse_panel/radio_morse_panel.c

```diff
@@ -22,139 +48,84 @@ void unit()
 	}
 }
 
-void dot()
+void PlayDot()
 {
 	carrier = CARRIER_ON;
-	unit();
+	PlayUnit();
 	carrier = CARRIER_OFF;
 }
 
-void dash()
+void PlayDash()
 {
 	carrier = CARRIER_ON;
-	unit(); unit(); unit();
+	PlayUnit(); PlayUnit(); PlayUnit();
 	carrier = CARRIER_OFF;
 }
 
-void short_space()
+void PlayShortSpace()
 {
-	unit();
+	PlayUnit();
 }
 
-void letter_space()
+void PlayLetterSpace()
 {
-	unit(); unit(); unit();
+	PlayUnit(); PlayUnit(); PlayUnit();
 }
 
-void word_space()
+void PlayWordSpace()
 {
-	unit(); unit(); unit(); unit(); unit(); unit(); unit();
+	PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit();
 }
 
-void zero()
-{
-	dash(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void one()
-{
-	dot();  short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void two()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void three()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void four()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dash();
-}
-
-
-void five()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot();
-}
-
-void six()
-{
-	dash(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot();
-}
-
-void seven()
-{
-	dash(); short_space();
-	dash(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot();
+unsigned char GetLetterPositionInMorseAlphabet(char Letter, const MorseCode *MorseAlphabet, const unsigned char NumOfElements){
+	unsigned char LetterPosition;
+	
+	// simply compare each element in alphabet;
+	for(LetterPosition = 0; LetterPosition < NumOfElements; LetterPosition++){
```

Comment:
```
	for(int i = 0; morseCode[i].symbol != 0; ++i) {
```
p.s. Don't comment easy-to-understand things. Function name must tell what this function does.